avocado/core/runner.py

```diff
@@ -359,6 +359,16 @@ def sigtstp_handler(signum, frame):     # pylint: disable=W0613
 
         signal.signal(signal.SIGTSTP, sigtstp_handler)
 
+        # Saving the original sys.path state
+        _syspath = sys.path[:]
+
+        # Injecting the test module into the sys.path, so user can define
+        # a method from the test module to be used in func_at_exit
+        if 'modulePath' in test_factory[1]:
+            test_path = test_factory[1]['modulePath']
+            test_module_dir = os.path.abspath(os.path.dirname(test_path))
+            sys.path.insert(0, test_module_dir)
```

Comment:
I'd like to avoid telling test writers to use pickle for this to work. Is way easier for then just to set the PYTHONPATH before running this test. Let me try something else. I get back to you soon.